### PR TITLE
Create notifications.liquid

### PR DESCRIPTION
**Pull request by diana-lakatos**
Notifications concept topic for the Get Started/Notifications section.

### CODE REVIEW

File: marketplace_builder/pages/getting-started/notifications/notifications.liquid [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+| `enabled`           | Boolean which defines whether alert should be invoked or not. If false, it will be simply ignored.                                                                                                                 | true    | false                                                                                                                                                                        |
+| `trigger_condition` | Liquid condition to control whether notification should be sent or not. The only value that allows notification to be delivered is `true`. If you skip it, the notification will be sent (unless enabled is false) | true    | {% raw %}`{% if should_send == 'some_value'%}true{% endif %}`{% endraw %} - this code will trigger the notification only if value of `should_send` is equal to 'some_value'. |
+
+{% include alert/important content="`to` property is required in SMS and API Call notifications." %}
```
Comment (pavelloz) [inline marketplace_builder/pages/getting-started/notifications/notifications.liquid]: {% include 'alert/important' content: "`to` property is required in SMS and API Call notifications." %}

File: marketplace_builder/pages/getting-started/notifications/notifications.liquid [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+| ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `to`                | Depending on the alert type, it is either: {::nomarkdown}<ul><li>an email address of the recipient</li><li>mobile number of the recipient</li><li>endpoint for the API call</li></ul>{:/} Accepts liquid.          | n/a     | `test@email.com`                                                                                                                                                             |
```
Comment (pavelloz) [inline marketplace_builder/pages/getting-started/notifications/notifications.liquid:36]: `{::nomarkdown}` will need to be verified - im positive its available in jekyll, buti im not sure its available in our markdown implementation

File: marketplace_builder/pages/getting-started/notifications/notifications.liquid (hunk 1 above)
Comment (diana-lakatos) [reply marketplace_builder/pages/getting-started/notifications/notifications.liquid]: Thank you, fixed it!

File: marketplace_builder/pages/getting-started/notifications/notifications.liquid (hunk 2 above)
Comment (diana-lakatos) [reply marketplace_builder/pages/getting-started/notifications/notifications.liquid:36]: @piotrze Could you please verify if `{::nomarkdown}` is available in our markdown implementation? Thanks!

**Comment by pavelloz** [comment]
I guess we will find out soon ;)

**Pull request merged**